misc/write_fake_manifests.py

```diff
@@ -76,7 +76,7 @@ def path(self):
 
     def src_obj_pairs(self, path, name):
         num_sources = paretoint(55, alpha=2) + 1
-        return [(os.path.join('..', '..', path, s + '.cc'),
+        return [(os.path.join('src', path, s + '.cc'),
```

Comment:
Yes, I'm just busy at the moment and suspend my ninja related activities.
Will pick this up again in a few days hopefully.

On Wed, Apr 6, 2016, 03:44 Nico Weber <notifications@github.com> wrote:

> In misc/write_fake_manifests.py
> <https://github.com/ninja-build/ninja/pull/1109#discussion_r58641323>:
>
> > @@ -76,7 +76,7 @@ def path(self):
> >
> >      def src_obj_pairs(self, path, name):
> >          num_sources = paretoint(55, alpha=2) + 1
> > -        return [(os.path.join('..', '..', path, s + '.cc'),
> > +        return [(os.path.join('src', path, s + '.cc'),
>
> (It's my impression that this isn't blocked on you waiting for me; is that
> correct?)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ninja-build/ninja/pull/1109/files/776a43047c076561777b5e9bd2635ce32c4f0104#r58641323>
>